src/custom-element.js

```diff
@@ -1964,6 +1964,7 @@ function createBaseCustomElementClass(win, elementConnectedCallback) {
       const resourceState = this.getResource_().getState();
       // Do not show fallback before layout
       if (
+        !this.R1() &&
```

Comment:
Why is this change necessary?

Reply:
This is because R1 components don't use the resource system (there was a thread with @dvoytenko on this in our Bento eng)

Reply:
Does this change require unit tests?

Reply:
Done.

Reply:
I'm also confused why this is needed